tests/virt/node/migration_and_maintenance/test_multifd_policy_behavior.py

```
@@ -0,0 +1,53 @@
+        self,
+        vm_for_multifd_test,
+        migrated_vm_source_pod,
+    ):
+        assert_multifd_capability_in_logs(
+            source_pod=migrated_vm_source_pod, expected_in_log=True, scenario_name="baseline scenario"
+        )
+
+    @pytest.mark.polarion("CNV-12265")
+    def test_multifd_disabled_with_cpu_limit(
+        self,
+        vm_for_multifd_test,
```

Ruff (0.12.2)

Unused method argument: `vm_for_multifd_test`

(ARG002)